### ISSUE TITLE
Some ocean variables with areacella in their cell_measures

### ISSUE THREAD

**Issue by matthew-mizielinski** [cmip6dr/CMIP6_DataRequest_VariableDefinitions#300]
Our science QA review of cmorized output has thrown up a few ocean variables which have `area: areacella` as their cell_measures. The code below can be used to pick them out from the data request;

```
for uid, cmorvar in dq.inx.CMORvar.uid.iteritems():
    if cmorvar.modeling_realm == 'ocean':
        # Cell measures are attached to the structure that the cmorvar links to
        cm = dq.inx.uid[cmorvar.stid].cell_measures
        # check for areacello and ignore some edge cases
        if 'areacello' not in cm and cm not in  ['', '--OPT', '--UGRID']:
            print '{0.mipTable}/{0.label} : {1}'.format(cmorvar, cm)
```
For version 01.00.21 this yields
```
Eday/mlotst : area: areacella
Emon/thetaot700 : area: areacella
3hr/tos : area: areacella
Emon/thetaot2000 : area: areacella
Emon/thetaot300 : area: areacella
Eday/t20d : area: areacella
```
I think all of these should have `area: areacello` as their cell_measures.

**Comment by martinjuckes** [comment]
Yes .. fixed in 01.00.22 .. thanks for pointing it out.

**Comment by martinjuckes** [comment]
An I've checked that the problem does not occur in realm 'ocnBgChem'
